libs/s25main/addons/const_addons.h

```diff
@@ -65,6 +65,8 @@ ENUM_WITH_STRING(AddonId, LIMIT_CATAPULTS = 0x00000000, INEXHAUSTIBLE_MINES = 0x
 
                  NUM_SCOUTS_EXPLORATION = 0x00C00000,
 
+                 AUTOFLAGS = 0x00900009,
```

Comment:
See the comment at the start: The value should be `0x00F00000` as it is your first addon